usaspending_api/etl/tests/integration/test_load_to_from_delta.py

```diff
@@ -806,12 +870,23 @@ def test_load_table_to_from_delta_for_recipient_testing(spark, s3_unittest_data_
 
 
 @mark.django_db(transaction=True)
-def test_load_table_to_from_delta_for_recipient_profile(spark, s3_unittest_data_bucket, hive_unittest_metastore_db):
-    baker.make("recipient.RecipientProfile", id="1", _fill_optional=True)
-    baker.make("recipient.RecipientProfile", id="2", _fill_optional=True)
-    _verify_delta_table_loaded_to_delta(spark, "recipient_profile", s3_unittest_data_bucket)
-    _verify_delta_table_loaded_from_delta(spark, "recipient_profile", spark_s3_bucket=s3_unittest_data_bucket)
-    _verify_delta_table_loaded_from_delta(spark, "recipient_profile", jdbc_inserts=True)  # test alt write strategy
+def test_load_table_to_from_delta_for_recipient_profile(
```

Comment:
In order for this test to actually verify that the results from the new Spark ETL match the Postgres ETL you would need to call the recipient_profile SQL. Currently this is only going to verify that the Recipient Profile in Spark matches with the mocked data.

The Transaction and Award Search queries did this naturally since they are Views, but I believe for the purpose of these tests and to get the same behavior the Postgres table needs to be re-generated.